spec/spec_helper.rb

```diff
@@ -11,6 +11,9 @@
 $LOAD_PATH.unshift File.expand_path('../..', __FILE__)
 
 RSpec.configure do |config|
+  require_relative './support/global_helpers.rb'
```

Comment:
You should be able to use `require` here as the `$LOAD_PATH` is modified to include the `spec` folder (see line 11).